packages/quantic/README.md

```diff
@@ -42,11 +42,27 @@ Optionally install the [VSCode Salesforce Extension Pack](https://marketplace.vi
 - You can also run this command from the command line. `sf org login web --set-default-dev-hub --alias LWC-Hub`
 - Running this command opens a browser to the Salesforce login page. Enter your Salesforce username and password. Authorize only once, not every time you work on your project.
 
-### Single command run
+### Single Command Run
 
 - Make sure you are at the root of `packages/quantic`.
-- Run `npm run scratch:dev`.
-- You should now have a scratch org with Quantic and the test community deployed. Watch your commandline for the url to the examples community.
+
+- Run `npm run setup:examples`.
+
+This command will create two scratch orgs for you:
+
+- One with Lightning Web Security (LWS) enabled.
+
+- One with LWS disabled.
```

Comment:
Suggestion:

```suggestion
- One with Lightning Web Security (LWS) enabled. (Alias: `Quantic__LWS_enabled`)

- One with LWS disabled, Locker Service enabled. (Alias: `Quantic__LWS_disabled`)
```

Also I think at some point I think we should rename those to be something like

`Quantic__LWS` and `Quantic__Locker`, it would represent better what we're testing.

(But not as part of this PR)